plugins/hls-tactics-plugin/src/Wingman/StaticPlugin.hs

```diff
@@ -19,9 +19,16 @@ staticPlugin :: DynFlagsModifications
 staticPlugin = mempty
   { dynFlagsModifyGlobal =
       \df -> allowEmptyCaseButWithWarning
+           $ flip gopt_unset Opt_SortBySubsumHoleFits
            $ df
+             { refLevelHoleFits = Just 0
+             , maxRefHoleFits   = Just 0
+             , maxValidHoleFits = Just 0
```

Comment:
is this the correct set of options? I would think what you need is just:
```
disableHoleFits :: DynFlags -> DynFlags
disableHoleFits = flip gopt_unset Opt_ShowValidHoleFits
```

Reply:
That would be the equivalent of saying we don't have a preference, right? Perhaps that's equivalent to getting zero fits, but if so, it's only by accident.

Reply:
Maybe not; I don't know. Whatever the situation, this gives the right result on my machine.

Reply:
I believe it means "disable all hole fit suggestions"